### ISSUE TITLE
Failing to load en_us, resorts to en_us(fallback) for all mods

### ISSUE THREAD

**Issue by jdstankosky** [CmdrJane/Discordium#7]
```
Failed to load default en_us locale for mod (everyModNameHere), trying to load default en_us locale
Loaded language en_us(fallback) for mod (everyModNameHere)
```

**Comment by CmdrJane** [comment]
1.18.1 or 1.18.2?

**Comment by jdstankosky** [comment]
I should have included that, sorry, 1.18.2

**Comment by CmdrJane** [comment]
not every mod should have localization files so as long as death and advancement messages works it should be fine
